app/hotels/src/filter/score/ScorePopup.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { Text } from 'react-native';
+import { ButtonPopup, Slider } from '@kiwicom/react-native-app-common';
+
+type Props = {|
+  onClose: () => void,
+  onSave: (minScore: number | null) => void,
+  isVisible: boolean,
+  minScore: number | null,
+|};
+
+type State = {|
+  minScore: number,
+|};
+
+const SLIDER_SHIFT = 5;
+
+export default class ScorePopup extends React.Component<Props, State> {
+  state = {
+    minScore: 0,
```

Comment:
This name is misleading, because this variable contains the slider value, ie. not the score, but the shifted score. Not sure how to rename it, maybe shiftedScore or sliderPosition.